assets/index.html

```diff
@@ -1,165 +1,164 @@
+  <style type="text/css">
+    body {
+      margin: 0;
+      padding: 0;
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+        "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+        sans-serif;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+    }
+
+    code {
+      font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+        monospace;
+    }
+
+    pre {
+      margin: 0px 0px;
+    }
+
+    @font-face {
+      font-family: 'Monument';
+      src: url("/monument-regular.otf") format("opentype");
+      font-weight: 400;
+    }
+
+    @font-face {
+      font-family: 'Monument';
+      src: url("/monument-medium.woff") format("woff");
+      font-weight: 450;
+    }
+
+    @font-face {
+      font-family: 'Monument';
+      src: url("/monument-medium.woff") format("woff");
+      font-weight: 500;
+    }
+
+    @font-face {
+      font-family: 'Monument';
+      src: url("/monument-bold.woff") format("woff");
+      font-weight: 500;
+    }
+
+    @font-face {
+      font-family: 'Monument';
+      src: url("/monument-regular-italic.woff") format("woff");
+      font-weight: 400;
+      font-style: italic;
+    }
+
+    @font-face {
+      font-family: 'Monument';
+      src: url("/monument-medium-italic.woff") format("woff");
+      font-weight: 500;
+      font-style: italic;
+    }
+
+    @font-face {
+      font-family: 'Monument';
+      src: url("/monument-bold-italic.woff") format("woff");
+      font-weight: 600;
+      font-style: italic;
+    }
+
+    @font-face {
+      font-family: 'Monument Semi-Mono';
+      src: url("/ABCMonumentGroteskSemi-Mono-Regular.woff") format("woff");
+      font-weight: 400;
+    }
+
+    @font-face {
+      font-family: 'Monument Semi-Mono';
+      src: url("/ABCMonumentGroteskSemi-Mono-Medium.woff") format("woff");
+      font-weight: 500;
+    }
+
+    @font-face {
+      font-family: 'Monument Semi-Mono';
+      src: url("/ABCMonumentGroteskSemi-Mono-Heavy.woff") format("woff");
+      font-weight: 600;
+    }
+
+    @font-face {
+      font-family: 'Monument Mono';
+      src: url("/ABCMonumentGroteskMono-Regular.woff") format("woff");
+      font-weight: 400;
+    }
+
+    @font-face {
+      font-family: 'Monument Mono';
+      src: url("/ABCMonumentGroteskMono-Medium.woff") format("woff");
+      font-weight: 500;
+    }
+
+    @font-face {
+      font-family: 'Monument Mono';
+      src: url("/ABCMonumentGroteskMono-Heavy.woff") format("woff");
+      font-weight: 600;
+    }
+  </style>
+  <!-- Fix Hljs background color so it can be overridden -->
+  <style>
+    .hljs {
+      background: unset;
+    }
+  </style>
+  <style>
+    html,
+    body {
+      background-color: #171a21;
+    }
+
+    html[data-theme-mode='light'],
+    html[data-theme-mode='light'] body {
+      background-color: #FFFFFF;
```

Comment:
this is the only line that actually changed, the rest is due to the formatter but won't happen in future commits